lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -202,6 +270,30 @@ class RenderForm extends React.Component {
         supportEmail={this.props.context.supportEmail}
       />);
     }
+    if (suggestions.length) {
+      suggestionsListComponent = (
+        <ul className="suggestions">
+          {suggestions.map((suggestion, index) => {
+            let className;
+
+            if (index === activeSuggestion) {
+              className = 'suggestion-active';
+            }
```

Comment:
move this if in return. This way, you would not need a variable just to store the classname.